### ISSUE TITLE
Training time and performance

### ISSUE THREAD

**Issue by kwea123** [NVIDIAGameWorks/kaolin-wisp#2]
Thanks for opening source such a huge & great project!
I would like to compare to other implementations, do you have any benchmarks on public datasets (e.g. synthetic-nerf)?
Btw, this is what I get from 5 minutes of training on fox using this command:
```
python3 app/main_interactive.py --config configs/ngp_nerf_interactive.yaml --multiview-dataset-format standard --mip 0 --dataset-path ../instant-ngp/data/nerf/fox/
```
![Screenshot from 2022-08-01 11-40-49](https://user-images.githubusercontent.com/11364490/182061987-d9bd931b-9e8f-4577-9b9c-5cc02f2d068d.png)

I don't see any PSNR on the graph, but judging from the visual quality and the loss, I reckon it is far less good than instant-ngp of the same training time. Is it to be expected?

Just saw your teaser image, it took 26 minutes to get to visually good quality as shown?

**Comment by tovacinni** [comment]
Hi @kwea123, thanks for your interest in our work! We don't have full benchmarks yet but that's on our TODO list in the coming weeks along with PSNR visualization on the GUI. 

The much slower training time compared to `instant-ngp` is very much expected; our code is mostly PyTorch based and selectively CUDA-optimized, although there's a few places we plan to optimize for better results. 

Just looking at your attached screenshots though there's something you can adjust to increase the visual quality. Right now the sampling of points to integrate along the ray is selected based on the near and far plane, and your far plane on the screenshot is set to 100. Decreasing this will significantly improve results... or alternatively you can choose the exact camera (with the near and far plane used in training) by choosing a camera on the GUI. 

Happy to answer whatever other questions you might have, or even discuss. 😁

**Comment by tovacinni** [comment]
For reference, here's an image after similar number of epochs as your screenshot.

![image](https://user-images.githubusercontent.com/2717105/182070797-f5165765-21bd-45f5-b355-8274cda2bea4.png)
